include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -689,6 +693,50 @@ typedef struct OrtApiBase OrtApiBase;
  */
 ORT_EXPORT const OrtApiBase* ORT_API_CALL OrtGetApiBase(void) NO_EXCEPTION;
 
+typedef struct OrtMetaDef {
+  const char* name;
+  const char* domain;
+  int since_version;
+
+  const char** inputs;
+  size_t input_len;
+  const char** outputs;
+  size_t output_len;
+  const char** constant_initializers;
+  size_t initializer_len;
+
+  const char* doc_string;
+} OrtMetaDef;
+
+typedef struct OrtIndexedSubGraph {
+  OrtMetaDef* meta_def; // TODO(leca): how to define a nested structure pointer?
```

Comment:
Does this have to be a pointer to an OrtMetaDef? It may simplify use if this is contained by value instead.

Reply:
It looks we will check the pointer is null or not to distinguish between single node mode and fused node mode (See base class IExecutionProvider::GetCapability() which does not set this pointer and TryAssignSingleNode() which will check this pointer)